quickwit-doc-mapper/src/tokenizers.rs

```diff
@@ -48,3 +141,135 @@ fn raw_tokenizer_test() {
     assert!(!haiku_stream.advance());
     assert!(!tokenizer.token_stream(my_long_text).advance());
 }
+
```

Comment:
Nice tests.

Reply:
Those aren't mine, they were already there.

Reply:
I was talking about the others on the log tokenizer.